php/modules/Session.php

```diff
@@ -99,11 +112,18 @@ public function login($username, $password) {
 		$username = crypt($username, $this->settings['username']);
 		$password = crypt($password, $this->settings['password']);
 
-		# Check login with crypted hash
 		if ($this->settings['username']===$username&&
 			$this->settings['password']===$password) {
 				$_SESSION['login']		= true;
 				$_SESSION['identifier']	= $this->settings['identifier'];
+
+				$expire = time() + 60 * $this->settings['sessionLength'];
```

Comment:
Yeah, the update script has to be synced up with a release version number. Something for electerious to coordinate if he merges and makes a release. Good to see the script in your repo to make it as easy as possible for him to pick up this patch.